src/nrnoc/cabcode.cpp

```diff
@@ -711,7 +704,9 @@ static Section* Sec_access(void) /* section symbol at pc */
         hoc_thisobject = 0;
         hoc_symlist = hoc_top_level_symlist;
     }
-    nrn_chk_section(s);
+    if (s->type != SECTION) {
```

Comment:
It was only being used once, in the file, and no where else is doing the same test; based on the name, it wasn't clear it would call `execerror` and thus exit the function early.  I can put it back, no problem.